### ISSUE TITLE
[P1][Sourcing][Accessibility][Tree table]The role and status of the checkboxes in the first column are not perceptible with the screen reader. Instead, JAWS only outputs the information that the cell contains controls. However, there is no way to get the information about the controls. The problem also occurs with JAWS 2023 and NVDA.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10278]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The role and status of the checkboxes in the first column are not perceptible with the screen reader. Instead, JAWS only outputs the information that the cell contains controls. However, there is no way to get the information about the controls. The problem also occurs with JAWS 2023 and NVDA.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
https://sap.github.io/fundamental-ngx/#/platform/table/basic#tree

#### If this is a bug, please provide steps for reproducing it.
1) Reach the first cell of the tree table using tab or arrow keys
2) Observe that screenreader does not announce checkbox role and its state

#### Is there anything else we should know?
Reported by customer

**Comment by droshev** [comment]
fixed with `0.43.8`

**Comment by sagar7259** [comment]
Hello,

As per the retest activity the issue is still not fixed. On accessing the checkboxes within the tree table the UI type check box or any information which specifies that there is a checkbox should be read.
On navigating through the first column of tree table below speech is read:

"TreeGrid
column header
Select all
column 1 row 1
Column 1 of 5, row: 1 of 6

1 of 5
Row 2
Column 1 of 5, row: 2 of 6

2 of 5
Row 3
Column 1 of 5, row: 3 of 6"

And even providing 'Aria-selected' to the selected rows would not be enough as this may confuse users between the radio button and checkbox. To resolve this confusion and this issue, UI type "Check box" should be associated on accessing them.

@rvamudha please reopen this issue.

Regards,
Sagar

**Comment by sagar7259** [comment]
@nikolay-kolarov 
The checkbox is read only using Virtual PC cursor mode and not with the forms mode.
Kindly look into it.

Regards,
Sagar

**Comment by sagar7259** [comment]
Hi Team,

The reported issue is now fixed, has the state "Selected" is being read correctly to the users and also the tutor message is correct. Hence we can close this ticket.

https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic

Regards,
Sagar
